TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -231,5 +259,55 @@ public class IncompatibleModsPanel : UIPanel {
                 ValueAnimator.Animate("ModalEffect", delegate(float val) { blurEffect.opacity = val; }, new AnimatedFloat(1f, 0f, 0.7f, EasingType.CubicEaseOut), delegate() { blurEffect.Hide(); });
             }
         }
+
+        private bool DeleteLocalTMPE()
+        {
+            PluginInfo localTMPE = GetLocalTMPE();
+            try
+            {
+                Log._Debug("Attempt to Unload local TM:PE");
+                localTMPE.Unload();
```

Comment:
Theoretically you don't have to call `Unload();` because when you remove folder of plugin, `OnFileWatcherEventChanged` is invoked which calls `OnPluginFolderRemoved()` and that method calls `this.RemovePluginAtPath(path);` under the hood :wink:
Didn't tested :smile:

Reply:
I'll give it a try :)

Reply:
It worked. :)